### PR TITLE
button styled by Ameer

### PR DESCRIPTION
**Pull request by ameerkabir**

### CODE REVIEW

File: skin.css [hunk 1]
```diff
@@ -10,4 +10,41 @@ button.btn {
...
+  text-align: center;
+  text-decoration: none;
+  display: inline-block;
+  font-size: 16px;
+  margin-left: 14px;
+  
+}
+button{
+  -webkit-transition-duration: 0.4s; /* Safari */
+  transition-duration: 0.4s;
```
Comment (alismith-dev) [inline skin.css:27]: The animation is a nice touch! 👍

File: skin.css [hunk 2]
```diff
@@ -10,4 +10,41 @@ button.btn {
...
+button:hover{
+  background-color: white;
+  color: rgba(111, 0, 255, 0.589);
+
+}
+.btn--primary{
+  background-color: rgba(58, 3, 161, 0.815);
+  color: white;
+  padding: 17px 59px;
+  margin-left: 10px;
```
Comment (alismith-dev) [inline skin.css:39]: Not sure why you're overriding the padding & margin here - should maybe be in the `.btn` ruleset

File: skin.css [hunk 3]
```diff
@@ -10,4 +10,41 @@ button.btn {
...
+  transition-duration: 0.4s;
+}
+
+button:hover{
+  background-color: white;
+  color: rgba(111, 0, 255, 0.589);
+
+}
+.btn--primary{
+  background-color: rgba(58, 3, 161, 0.815);
```
Comment (alismith-dev) [inline skin.css:36]: These colours don't quite match - it looks a bit too purple

**Review by 40thieves — COMMENTED** [review]
A couple of comments, but on the right track! 👍 🎉 

The next steps would be to add some `<ul>`s and a Sass loop for paragraphs. See [here for more details](https://codeyourfuture.github.io/syllabus-london/html-css/lesson3.html#homework)

**Pull request closed without merging**